roles/mongodb_auth/molecule/default/tests/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+# defaults file for mongodb_auth
+mongod_port: 27017
+mongod_package: "mongodb-org-server"
+
+authorization: "enabled"
+
+# when adding auth, the login credentials to use
+mongodb_admin_user: admin
+# For production use - please change the admin password!
+mongodb_admin_pwd: adminchangedpassword
```

Comment:
This file should be put back to what it was. I have spotted the original problem.